### PR TITLE
Add Spanish translation L10n-Common.json

### PR DESCRIPTION
**Pull request by JuanVqz**
### Description
Add Spanish translate for [L10n-Common.json](https://github.com/mdn/content/blob/main/files/jsondata/L10n-Common.json)

### Motivation
Help fixing Spanish translations

### Related issues and pull requests

Relates to https://github.com/mdn/translated-content/issues/8749

### CODE REVIEW

File: files/jsondata/L10n-Common.json [hunk 1]
```diff
@@ -173,6 +174,7 @@
     "fr": "[Page qui reste à écrire]",
     "ja": "[ページがまだ書かれていません]",
     "pt-BR": "[Página ainda não foi escrita]",
-    "zh-CN": "[页面尚未撰写]"
+    "zh-CN": "[页面尚未撰写]",
+    "es": "[Página aún no escrita]",
```
Comment (JuanVqz) [inline files/jsondata/L10n-Common.json:178]:
```suggestion
    "es": "[Página aún no escrita]"
```

File: files/jsondata/L10n-Common.json [hunk 2]
```diff
@@ -37,7 +37,8 @@
     "pt-BR": "Handlers de Eventos",
     "zh-TW": "事件處理器",
     "zh-CN": "事件处理器",
-    "ru": "Обработчики событий"
+    "ru": "Обработчики событий",
+    "es": "Manejador de eventos"
```
Comment (bsmth) [inline files/jsondata/L10n-Common.json:41]: Could we make this in alphabetical order?

File: files/jsondata/L10n-Common.json [hunk 3]
```diff
@@ -173,6 +174,7 @@
     "fr": "[Page qui reste à écrire]",
     "ja": "[ページがまだ書かれていません]",
     "pt-BR": "[Página ainda não foi escrita]",
-    "zh-CN": "[页面尚未撰写]"
+    "zh-CN": "[页面尚未撰写]",
+    "es": "[Página aún no escrita]"
```
Comment (bsmth) [inline files/jsondata/L10n-Common.json:178]: Same as above

**Review by bsmth — APPROVED** [review]
Thanks @JuanVqz

**Pull request merged**